framework/src/task_manager/task_manager_interface.c

```diff
@@ -0,0 +1,603 @@
+/****************************************************************************
```

Comment:
1. taskmgr_interface.c is not deleted. What is this?
2. In my opinion, this is not an interface. 
taskmgr_receive_response and taskmgr_send_request are interface, not others.
Let's split them.

Reply:
Done